nevergrad/optimization/mutations.py

```diff
@@ -46,9 +46,9 @@ def portfolio_discrete_mutation(parent: ArrayLike, u: Optional[int] = None) -> A
     """
     dimension = len(parent)
     if u is None:
-        u = int(np.random.randint(1, dimension))
+        u = 1 if dimension == 1 else int(np.random.randint(1, dimension))
```

Comment:
@teytaud what is a sensible portfolio mutation for `dimension=1`? This used to raise an error, and I changed it to "mutate it everytime" but this may not be the best solution?

Reply:
The algorithm is not made for such a case, but yes, mutate every time looks like the best solution to me.
Yes, u=1 corresponds to mutate every time.